### ISSUE TITLE
Changing the functionality of FCM 

### ISSUE THREAD

**Issue by belalakbar** [policycompass/policycompass#130]
The fixed value tab will be removed and new function will be added in the algorithm, which automatically detects the fixed value. 

The advance setting for activator function will be provided.

**Comment by cbotsikas** [comment]
@aquinas9 why is this closed? The PR is still pending. @belalakbar can you review the PR?
https://github.com/policycompass/policycompass-frontend/pull/122

**Comment by aquinas9** [comment]
We will review the pull request! It happend just becasue the function has been implemented in bilal's localhost. we will check again.

**Comment by cbotsikas** [comment]
Is it possible to have it mergeable by Monday so that we can merge and tag the repositories with new version since sprint 2 is over, or shall we move this for Sprint 3?

There is also a PR for fcmmanager: https://github.com/policycompass/policycompass-fcmmanager/pull/10 
I guess it's related with the pending one on frontend?

**Comment by belalakbar** [comment]
This issue has been solved and changes made in both fcmmanager and frontend and I did PR on both for this.

**Comment by cbotsikas** [comment]
@belalakbar If you are referring to PRs https://github.com/policycompass/policycompass-frontend/pull/122 and https://github.com/policycompass/policycompass-fcmmanager/pull/10 the first one (frontend) is not mergable and you have to fix it. Otherwise i don't see any new PR. We need to tag the repositories ASAP, before we can start pushing/merging sprint 3 work. If you can't fix it in time, we'll move this issue to Sprint 3.

**Comment by FabiApfelkern** [comment]
@belalakbar Can we please merge this by today?

**Comment by belalakbar** [comment]
Yes Fabian merge, this functionality has already appear in the live server.

**Comment by cbotsikas** [comment]
@belalakbar how is this functionality in live server?? With which commits? Can you provide links?

There are two pending Pull Requests related with this issue.
One for the frontend, which *we can not merge* unless you fix it: https://github.com/policycompass/policycompass-frontend/pull/122
And one for the FCM manager here: https://github.com/policycompass/policycompass-fcmmanager/pull/10

**Comment by belalakbar** [comment]
issue 130 state that  "The fixed value tab will be removed and new function will be added in the algorithm, which automatically detects the fixed value."
 @cbotsikas the functionality on live server means, that on stage-policy compass this function is properly working as it required
